extension/src/dashboard/pages/Notifications/__tests__/NotificationsList.test.tsx

```diff
@@ -0,0 +1,213 @@
+        </UserProvider>
+      </Router>,
+    );
+
+    await mockAxiosResponse(
+      { data: { items: [], total: 0 } },
+    );
+
+    expect(getByText('Aucunes nouvelles notifications'));
+  });
+
+  it('should display fallback message if no seen notification', async () => {
+    history.push('/notifications/lues');
+
+    const { getByText } = render(
+      <Router history={history}>
+        <UserProvider value={{ user: mockUser, setUser: () => {} }}>
+          <NotificationsCountProvider>
+            <Notifications />
+          </NotificationsCountProvider>
+        </UserProvider>
+      </Router>,
+    );
+
+    await mockAxiosResponse(
+      { data: { items: [], total: 0 } },
+    );
+
+    expect(getByText('Aucunes notifications'));
+  });
+
+  it('should be on unseen notifications tab', async () => {
+    const { getByText } = render(
+      <Router history={history}>
+        <UserProvider value={{ user: mockUser, setUser: () => {} }}>
+          <NotificationsCountProvider>
+            <Notifications />
+          </NotificationsCountProvider>
+        </UserProvider>
+      </Router>,
+    );
+
+    await mockAxiosResponse(
+      { data: { items: [], total: 0 } },
+    );
+
+    expect(getByText('Non lues').parentElement).toHaveAttribute('aria-selected', 'true');
+  });
+
+  it('should set notification as seen on notification item click', async () => {
+    const { getByText } = render(
+      <Router history={history}>
+        <UserProvider value={{ user: mockUser, setUser: () => {} }}>
+          <NotificationsCountProvider>
+            <Switch>
+              <Route path="/notifications" component={Notifications} />
+              <Route path="/information" render={({ location }) => <>{ location.search }</>} />
+            </Switch>
+          </NotificationsCountProvider>
+        </UserProvider>
+      </Router>,
+    );
+
+    await mockAxiosResponse(
+      { data: mockedNotifications },
+    );
+
+    act(() => {
+      userEvent.click(getByText('a répondu à une réaction sur l\'information'));
+    });
+
+    expect(getByText('?notificationId=1')).toBeVisible();
+  });
+
+  it('should set notification as seen and remove it from the list on seen icon click', async () => {
+    const { getByTestId, queryByText } = render(
+      <Router history={history}>
+        <UserProvider value={{ user: mockUser, setUser: () => {} }}>
+          <NotificationsCountProvider>
+            <Notifications />
+          </NotificationsCountProvider>
+        </UserProvider>
+      </Router>,
+    );
+
+    await mockAxiosResponseFor(
+      { url: '/api/notification/me/count' },
+      { data: { count: 1 } },
+    );
+
+    await mockAxiosResponse(
+      { data: mockedNotifications },
+    );
+
+    act(() => {
+      userEvent.click(getByTestId('set-seen-icon'));
+    });
+
+    await mockAxiosResponseFor(
+      { method: 'POST', url: '/api/notification/1/seen' },
+      { data: undefined, status: 204 },
```

Comment:
Do we need to explicit `data: undefined`?

Reply:
Yes: 
 > Argument of type '{ status: number; }' is not assignable to parameter of type 'HttpResponse'.
  Property 'data' is missing in type '{ status: number; }' but required in type 'HttpResponse'.